# Generated for NaoQi method ALLeds.listGroups
# Lists available group names.

---
# A vector of group names.
string[] value


